app/controllers/api/v2/ansible_roles_controller.rb

```diff
@@ -32,28 +33,30 @@ def destroy
         process_response @ansible_role.destroy
       end
 
-      api :PUT, '/ansible_roles/import', N_('Import Ansible roles')
-      param :proxy_id, :identifier, :required => true, :desc => N_('Smart Proxy to import from')
-      param :role_names, Array, N_('Ansible role names to import')
+      api :PUT, '/ansible_roles/import', N_('Sync Ansible roles')
+      param :proxy_id, :identifier, :required => true, :desc => N_('Smart Proxy to sync from')
+      param :role_names, Array, N_('Ansible role names to synced')
       def import
-        @imported = @importer.import!(role_names)
+        params = @importer.import!(role_names)
+        task = @importer.confirm_import(params)
+        render json: {
+          task: task
+        }, status: :ok
       end
 
-      api :PUT, '/ansible_roles/obsolete', N_('Obsolete Ansible roles')
+      api :PUT, '/ansible_roles/obsolete', N_('DEPRECATED: Obsolete Ansible roles')
```

Comment:
Please, use
```ruby
, deprecated: true
```
option when specifying deprecated endpoints.

I've noticed this should be added to few more places.